sites/engineering/features/content.feature

```diff
@@ -0,0 +1,277 @@
+#  When I fill in "Search faculty by name" with "Cohen"
+#  And I press "Go"
+#  Then I should see "Margaret Cohen" in the "Content Body" region
+
+#Scenario: Courses page content
+#  Given I am on "courses"
+#  Then I should see the heading "Current Courses" in the "Content" region
+
+#Scenario: Degree Programs page content
+#  Given I am on "degree-programs/overview-degree-programs"
+#  Then I should see "Stanford’s English curriculum features a team-taught, yearlong core sequence" in the "Content Body" region
+
+#Scenario: Department Bookshelf page content
+#  Given I am on "bookshelf"
+#  Then I should see the heading "Faculty Publications" in the "Content Body" region
+
+#Scenario: News page content
+#  Given I am on "news"
+#  Then I should see the heading "Wilfred Stone, professor emeritus of English passes away at 97" in the "Content" region
+
+#Scenario: Events page content
+#  Given I am on "events"
+#  Then I should see the heading "Upcoming Events" in the "Content Body" region
+#  And I should see "View Past Events" in the "Content Body" region
+
+@safe
+Scenario: Verify that the magnifying glass is visible
+  Given I am on the homepage
+  Then I should see a "#block-stanford-search-api-search-api-search-block" element
+  And I should see the "#block-stanford-search-api-search-api-search-block" element in the "Main Navigation" region
+
+#Figure out which HTML element has the click function for "When I click..."
+#Scenario: Verify that clicking on the magnifying glass gives user the search box
+#  Given I am on the homepage
+#  When I click on the element with css selector ".form-item"
+#  Then I should see the ".contextual-links-trigger" element in the "#block-stanford-search-api-search-api-search-block" region
+
+#Find another scenario that has search function or input field
+
+#Scenario: Verify that the search function for "internet" works
+#  Given I am on the homepage
+#  When I fill in "#edit-keywords" with "internet"
+#  #And I press the "enter" key in the ".input-medium" field
+
+@safe
+Scenario: Verify that the homepage has the correct headings
+  Given I am on the homepage
+  Then I should see the heading "About the School" in the "Content 3 column flow" region
+  And I should see the heading "Recent News" in the "Content 3 column flow" region
+  And I should see the heading "Research" in the "Content 3 column flow" region
+  And I should see the heading "For Students" in the "Content Lower" region
+
+@safe
+Scenario: Verify that the search results are via Solr and not Drupl
+  Given I am on "/search"
+  Then I should not see an "p[class='search-info']" element
+
+@safe
+Scenario: Verify that the full width banner at the top of the page is visibile
+  Given I am on the homepage
+  Then I should see the "img" element in the "Top Full Width" region
+
+@safe
+Scenario: Verify that the Stanford footer logo image is visible
+  Given I am on the homepage
+  Then I should see the "img" element in the "Footer" region
+
```

Comment:
Any way to make this test more specific? That is, look for something that will identify the logo, rather than just an image?